gooddata-sdk/README.md

```diff
@@ -30,16 +47,13 @@ for row in table.read_all():
     print(row)
 ```
 
-## Documentation
-
-Documentation is available [here](https://gooddata-sdk.readthedocs.io)
 
-## Bugs/Requests
+## Bugs & Requests
```

Comment:
Should be the same in `README` files `gooddata-sdk`, `gooddata-pandas` and `gooddata-fdw`

Reply:
fix'd